website/docs/r/mount.html.md

```diff
@@ -19,6 +19,25 @@ resource "vault_mount" "example" {
 }
 ```
 
+```hcl
+resource "vault_mount" "kvv2-example" {
+  path        = "version2-example"
+  type        = "kv-v2"
+  description = "This is an example KV Version 2 secret engine mount"
+}
+```
+
+```hcl
+resource "vault_mount" "transit-example" {
+  path                      = "transit-example"
+  type                      = "transit"
+  options = {
+    convergent_encryption = false
+  }
+  description = "This is an example transit secret engine mount"
+}
```

Comment:
Nit: the spacing seems a little off for the path and type fields. Maybe something like this instead?
```suggestion
resource "vault_mount" "transit-example" {
  path        = "transit-example"
  type        = "transit"
  description = "This is an example transit secret engine mount"

  options = {
    convergent_encryption = false
  }
}
```